src/actions/admin.js

```diff
@@ -465,13 +465,13 @@ export function uploadPlugin(fileData: File, force: boolean = false): ActionFunc
     };
 }
 
-export function installPluginFromUrl(url: string, force: boolean = false): ActionFunc {
+export function installPluginFromUrl(url: string, signatureUrl: ?string = null, force: boolean = false): ActionFunc {
```

Comment:
Just curious, what is the purpose of `?` in `?string`? Given that is a middle param, is it possible to completely omit?

Reply:
`?string` means that the parameter can be `null`. Actually with the new changes it might be possible yes.

Reply:
actually i think it makes sense to allow null because it's an optional parameter.

Reply:
Thanks! I thought that it could also mean to omit the parameter.

Reply:
👌 any parameter that has a default value can be omitted.